openblas/src/main/java/org/bytedeco/openblas/presets/openblas_nolapack.java

```diff
@@ -136,43 +137,79 @@ public class openblas_nolapack implements LoadEnabled, InfoMapper {
-            "clangb", "dlangb", "slangb", "zlangb", "ctrsyl3", "dtrsyl3", "strsyl3", "ztrsyl3",
-            "ctz_trans", "dtz_trans", "stz_trans", "ztz_trans", "ctz_nancheck", "dtz_nancheck", "stz_nancheck", "ztz_nancheck",
-            "sgedmd", "dgedmd", "cgedmd", "zgedmd", "sgedmdq", "dgedmdq", "cgedmdq", "zgedmdq", "sorhr_col", "dorhr_col", "cunhr_col", "zunhr_col",
+            "openblas_set_num_threads", "goto_set_num_threads", "openblas_get_num_threads",
+            "openblas_get_num_procs",
+            "openblas_get_config", "openblas_get_corename", "openblas_get_parallel",
+            "cblas_cdotc", "cblas_cdotu", "cblas_cimatcopy", "cblas_comatcopy", "cblas_dimatcopy",
+            "cblas_domatcopy", "cblas_cgeadd", "cblas_dgeadd", "cblas_sgeadd", "cblas_simatcopy",
+            "cblas_somatcopy", "cblas_zdotc", "cblas_zdotu", "cblas_zgeadd", "cblas_zimatcopy",
+            "cblas_zomatcopy", "clacrm", "dlacrm", "slacrm", "zlacrm", "clarcm", "dlarcm", "slarcm",
+            "zlarcm", "classq", "dlassq", "slassq", "zlassq",
+            "lapack_make_complex_double", "lapack_make_complex_float",
+            "cgetsqrhrt", "dgetsqrhrt", "sgetsqrhrt", "zgetsqrhrt",
+            "dorgtsqr_row", "sorgtsqr_row", "cungtsqr_row", "zungtsqr_row",
+            "ctz_trans", "dtz_trans", "stz_trans", "ztz_trans",
+            "ctz_nancheck", "dtz_nancheck", "stz_nancheck", "ztz_nancheck",
+            "sorhr_col", "dorhr_col", "cunhr_col", "zunhr_col",
             // deprecated
-            "cgegs",   "cggsvd",  "ctzrqf",  "dgeqpf",  "dlatzm",  "sgelsx",  "slahrd",  "zgegv",   "zggsvp",
-            "cgegv",   "cggsvp",  "dgegs",   "dggsvd",  "dtzrqf",  "sgeqpf",  "slatzm",  "zgelsx",  "zlahrd",
-            "cgelsx",  "clahrd",  "dgegv",   "dggsvp",  "sgegs",   "sggsvd",  "stzrqf",  "zgeqpf",  "zlatzm",
-            "cgeqpf",  "clatzm",  "dgelsx",  "dlahrd",  "sgegv",   "sggsvp",  "zgegs",   "zggsvd",  "ztzrqf",
-            // extended
-            "cblas_sbstobf16", "cblas_sbdtobf16", "cblas_sbf16tos", "cblas_dbf16tod", "cblas_sbdot", "cblas_sbgemv", "cblas_sbgemm", "cblas_sbgemm_batch",
-            "cgbrfsx", "cporfsx", "dgerfsx", "sgbrfsx", "ssyrfsx", "zherfsx", "cgerfsx", "csyrfsx", "dporfsx", "sgerfsx", "zgbrfsx", "zporfsx",
-            "cherfsx", "dgbrfsx", "dsyrfsx", "sporfsx", "zgerfsx", "zsyrfsx", "cgbsvxx", "cposvxx", "dgesvxx", "sgbsvxx", "ssysvxx", "zhesvxx",
-            "cgesvxx", "csysvxx", "dposvxx", "sgesvxx", "zgbsvxx", "zposvxx", "chesvxx", "dgbsvxx", "dsysvxx", "sposvxx", "zgesvxx", "zsysvxx"};
+            "cgegs", "cggsvd", "ctzrqf", "dgeqpf", "dlatzm", "sgelsx", "slahrd", "zgegv", "zggsvp",
+            "cgegv", "cggsvp", "dgegs", "dggsvd", "dtzrqf", "sgeqpf", "slatzm", "zgelsx", "zlahrd",
+            "cgelsx", "clahrd", "dgegv", "dggsvp", "sgegs", "sggsvd", "stzrqf", "zgeqpf", "zlatzm",
+            "cgeqpf", "clatzm", "dgelsx", "dlahrd", "sgegv", "sggsvp", "zgegs", "zggsvd", "ztzrqf"
```

Comment:
Please revert all unnecessary formatting changes. It makes it hard to compare what actually changed

Reply:
I've rewritten it now to match the original one as much as possible.